printer_mods/Boingomw/Wago_mount/README.md

```diff
@@ -0,0 +1,20 @@
+# Wago_Mounts
+Wago 221 Lever Nuts are really cool, and easy to use (and not inexpensive).  The 221-412 Lever Nut can be used to make your heater bed modular.  You can get them here: https://www.amazon.com/Lever-Nuts-Compact-Splicing-Connector-Contains/dp/B093LQSQXG/ref=psdc_6426215011_t3_B08W3QXN9B?th=1
+
+There are also very nice mounting brackets for them.  The one I based mine on was created by deepfriedheroin and is available here: https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/deepfriedheroin/v2_bed_wagos
+
+But, that mod assumes nothing else is going to be taking up that space.  I am going to also use the purge bucket and brush mod (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/edwardyeeks/Decontaminator_Purge_Bucket_%26_Nozzle_Scrubber), so I won't have enough area to take advantage of DeepFried's mount.  So I made a smaller one, that mounts flush to the same extrusion the Z-stop is on.
```

Comment:
Please make it a relative link.  That whole second sentence would be

    I am going to also use the [purge bucket and brush mod](../../edwardyeeks/Decontaminator_Purge_Bucket_%26_Nozzle_Scrubber), so I won't have enough area to take advantage of DeepFried's mount.

Reply:
done